blog/2022-09-20-spring-for-graphql-naming.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Spring for GraphQL is the recommended Spring integration"
+authors: andi
```

Comment:
put your name donna

Reply:
I wasn't sure if I had enough gravitas to make the announcement

Reply:
yep your name here :-)